diff -ruN fsspec-0.9.0-orig/fsspec.egg-info/PKG-INFO fsspec-0.9.0/fsspec.egg-info/PKG-INFO
--- fsspec-0.9.0-orig/fsspec.egg-info/PKG-INFO	2021-04-05 16:11:13.000000000 +0200
+++ fsspec-0.9.0/fsspec.egg-info/PKG-INFO	2021-04-15 10:31:30.000000000 +0200
@@ -94,6 +94,7 @@
 Classifier: Programming Language :: Python :: 3.9
 Requires-Python: >=3.6
 Description-Content-Type: text/markdown
+Provides-Extra: entrypoints
 Provides-Extra: abfs
 Provides-Extra: adl
 Provides-Extra: dask
diff -ruN fsspec-0.9.0-orig/fsspec.egg-info/requires.txt fsspec-0.9.0/fsspec.egg-info/requires.txt
--- fsspec-0.9.0-orig/fsspec.egg-info/requires.txt	2021-04-05 16:11:13.000000000 +0200
+++ fsspec-0.9.0/fsspec.egg-info/requires.txt	2021-04-15 10:31:30.000000000 +0200
@@ -1,7 +1,4 @@
 
-[:python_version < '3.8']
-importlib_metadata
-
 [abfs]
 adlfs
 
@@ -17,6 +14,11 @@
 requests
 dropbox
 
+[entrypoints]
+
+[entrypoints:python_version < "3.8"]
+importlib_metadata
+
 [gcs]
 gcsfs
 
diff -ruN fsspec-0.9.0-orig/PKG-INFO fsspec-0.9.0/PKG-INFO
--- fsspec-0.9.0-orig/PKG-INFO	2021-04-05 16:11:14.106077400 +0200
+++ fsspec-0.9.0/PKG-INFO	2021-04-15 10:31:30.246895600 +0200
@@ -94,6 +94,7 @@
 Classifier: Programming Language :: Python :: 3.9
 Requires-Python: >=3.6
 Description-Content-Type: text/markdown
+Provides-Extra: entrypoints
 Provides-Extra: abfs
 Provides-Extra: adl
 Provides-Extra: dask
diff -ruN fsspec-0.9.0-orig/setup.py fsspec-0.9.0/setup.py
--- fsspec-0.9.0-orig/setup.py	2021-04-05 16:10:42.000000000 +0200
+++ fsspec-0.9.0/setup.py	2021-04-15 10:30:09.000000000 +0200
@@ -35,7 +35,7 @@
     python_requires=">=3.6",
     install_requires=open("requirements.txt").read().strip().split("\n"),
     extras_require={
-        ":python_version < '3.8'": ["importlib_metadata"],
+        "entrypoints": ["importlib_metadata ; python_version < '3.8' "],
         "abfs": ["adlfs"],
         "adl": ["adlfs"],
         "dask": ["dask", "distributed"],
